src/main/java/seedu/address/logic/commands/WorkerPayCommand.java

```diff
@@ -0,0 +1,62 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.worker.Worker;
+
+
+
+/**
+ * Prints the pay earned by a worker identified using their displayed index.
+ */
+public class WorkerPayCommand extends Command {
+
+    public static final String COMMAND_WORD = "worker-pay";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Prints the pay earned by a worker identified by the index number used in the displayed worker list.\n"
```

Comment:
```suggestion
            + ": Calculates the weekly pay earned by a worker identified by the index number used in the displayed worker list.\n"
```
`Prints` seems to be a more programming-specific term which the target user (ie. a McDonald's manager) may not understand.